### DISCUSSION TITLE
Step size when navigating months

### DISCUSSION THREAD

**Discussion by pdhruv93** [gpbl/react-day-picker#1967] [Support] [answered]
I am using pagedNavigation with  numberOfMonths={2}. Is it possible to have the step size of 1 when navigating?
For example I have Mar and Apr. When clicking Next it shows May and June whereas I want to show Apr and May.
I am using v8.9.1

**Comment by gpbl** [comment, marked as answer]
@pdhruv93 if I understand correctly, `pagedNavigation` does the opposite you want to do. Turn `pageNavigation` back to `false` and it will step 1 month per click.

**Reply by pdhruv93** [reply]
Thanks, missed it completely.
